entity/src/versioned_purl.rs

```diff
@@ -27,6 +27,20 @@ pub enum Relation {
     PurlStatus,
 }
 
+pub struct AdvisoriesLink;
+
+impl Linked for AdvisoriesLink {
+    type FromEntity = Entity;
+    type ToEntity = super::purl_status::Entity;
+
+    fn link(&self) -> Vec<LinkDef> {
+        vec![
+            Relation::BasePurl.def(),
+            //super::purl_status::Relation::BasePurl.def(),
```

Comment:
If it's dead, sweep it under the rug.

Reply:
Remind me to be cautious if you ever acquire a Bob-sized rug...